src/Components/Components/src/Auth/AuthorizeView.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.AspNetCore.Authorization;
+
+namespace Microsoft.AspNetCore.Components
+{
+#pragma warning disable PUB0001 // Pubternal type in public API
```

Comment:
Because `.razor` files can't generate either `abstract` or `internal` yet, so this is the best I can do to discourage use.

Reply:
Is that in the roadmap?

Reply:
I remember seeing an issue for it somewhere, but it's not commited to any ETA.

Reply:
yeah it's not tracked for this release.